python/sglang/srt/tracing/trace.py

```diff
@@ -87,39 +79,41 @@ class SglangTraceThreadInfo:
     tracer: trace.Tracer
 
 
+@dataclass
+class SglangTraceEvent:
```

Comment:
```suggestion
class SGLangTraceEvent:
```
nit: we should probably use the correct uppercase and lowercase of SGLang.